api-reference/10 UI Widgets/Component/1 Configuration/onOptionChanged.md

```diff
@@ -24,3 +24,22 @@ The modified option if it belongs to the first level. Otherwise, the first-level
 The modified option's new value.
 
 ---
+
+The following example shows how to subscribe to a desired option of {WidgetName}:
+
+---
+##### jQuery
+
+    <!--index.js-->
+    $(function() {
+        $("#{widgetName}Container").{WidgetName}({
+            // ...
+            onOptionChanged: function(e) {
+                if(e.name == "desiredOptionName") {
+                    // ...
```

Comment:
1. I would use the name 'changedOption' and the strict equality
2. in the comment - // handle the option change here